common/source/docs/common-efi.rst

```diff
@@ -106,3 +109,22 @@ First, set either RPM1 or RPM2 sensor as being sourced from the EFI (examples wi
 
 - set :ref:`RPM1_TYPE<RPM1_TYPE>` = 3 (EFI)
 - then setup the :ref:`common-rpm-based-notch`
+
+Throttle Linearization
+======================
+
```

Comment:
do we need to mention about the AP_EFI_ThrLin Enable parameter?

Reply:
yes,will add that param